include/swift/AST/DiagnosticsParse.def

```diff
@@ -686,6 +686,18 @@ ERROR(sil_witness_assoc_conf_not_found,none,
 ERROR(sil_witness_protocol_conformance_not_found,none,
       "sil protocol conformance not found", ())
 
+// SIL differentiability witnesses
+ERROR(sil_diff_witness_expected_keyword,PointsToFirstBadToken,
+      "expected '%0' in differentiability witness", (StringRef))
+ERROR(sil_diff_witness_expected_parameter_list,PointsToFirstBadToken,
+      "expected an comma-separated list of parameter indices, e.g. (0, 1)", ())
```

Comment:
The syntax is no longer comma-separated or parenthesized.

Reply:
Fixed in https://github.com/apple/swift/pull/27487/commits/771143ae652d5b9cc49fc152863b4d133ee9c9c2:
```
expected a space-separated list of indices, e.g. '0 1'
```